go-controller/pkg/metrics/ovn.go

```diff
@@ -380,4 +433,6 @@ func RegisterOvnControllerMetrics() {
 	go ovnControllerConfigurationMetricsUpdater()
 	// ovn-controller coverage show metrics updater
 	go coverageShowMetricsUpdater(ovnController)
+	// ovn-controller subnet configuration metrics updater
```

Comment:
This is not really an "updater", more like a "setter" since this is only done once.

Reply:
Ok I will change.

Reply:
Done